package.json

```diff
@@ -161,6 +161,7 @@
     "@nodert-win10/windows.storage.streams": "^0.2.95",
     "appdmg": "^0.4.5",
     "dbus": "GPMDP/node-dbus#linux-only",
```

Comment:
Do we still need this `dbus` dep?

Reply:
Just checked and we do not. With the mpris change that got pulled in and now this it can be removed.